backend/clubs/views.py

```diff
@@ -5354,6 +5354,52 @@ def transfer(self, request, *args, **kwargs):
     def get_queryset(self):
         return Ticket.objects.filter(owner=self.request.user.id)
 
+    def _give_tickets(self, user, order_info, cart, reconciliation_id):
```

Comment:
Do we need to be passing in `reconciliation_id` when it's already part of `order_info`?